Pages/fashionCategoryPage.js

```diff
@@ -0,0 +1,31 @@
+class FashionCategoryPage{
+    constructor(){
+        this.title ="Fashion Category"
+    }
+    get brownJacket(){
```

Comment:
Have one getter for item and one method for clicking on item.
There should be parametar which will decide which item should be located/clicked.

Reply:
I added parametres for functions, so that we can define which  jacket we want to click in data.js, and in test we send data from js to method in page object.